tools/timedclient/timedclient.cpp

```diff
@@ -1482,6 +1482,10 @@ static void do_add_event(char *args)
     {
       cur_eve.setTimezone(val);
     }
+    else if( !strcmp(key, "MaximalTimeoutSnoozeCounter") )
```

Comment:
Maybe this should start with lower case like other event args?

Reply:
+1 (i was copying the info msg, ill change both)